apps/design-system-docs/docusaurus.config.ts

```diff
@@ -0,0 +1,129 @@
+import { themes as prismThemes } from 'prism-react-renderer';
+import type * as Preset from '@docusaurus/preset-classic';
+import type { Config } from '@docusaurus/types';
+
+// This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
+
+const config: Config = {
+	title: 'Automattic Design System',
+	tagline: 'TODO: Do we have a tagline?',
+	// TODO: Add a favicon
+	favicon: 'img/favicon.ico',
+
+	// TODO: Set the production url here
+	url: 'https://your-docusaurus-site.example.com',
```

Comment:
We'll need to figure this out in one of the next follow-ups:

- A production URL that's always up to date with the latest `trunk`
- Temporary URLs that allow testing the DS changes in every PR.

Reply:
Tracked in DS-151